index.js

```diff
@@ -1,15 +1,24 @@
+/*global navigator*/
+
 'use strict';
 
-const os = require('os');
 const pico = require('./lib/picomatch');
 
-const isWindows = os.platform() === 'win32';
+function isWindows() {
+  if (typeof navigator !== 'undefined' && navigator.platform) {
+    return navigator.platform.toLowerCase().indexOf('win') !== -1;
+  } else if (typeof process !== 'undefined' && process.platform) {
+    return process.platform.toLowerCase().indexOf('win') !== -1;
```

Comment:
Won't this match `darwin` too? I think this should strictly match `win32` since that's what `process.platform` specifies.